mantidproject/mantid
# Prevent the zoom being cleared on the instrument view after updating detectors

Currently any zooming applied to the view will be cleared when the user, for example, changes the x integration range. This can be annoying for users who trying to alter the range to enhance what they've zoomed in to look at.

**To test:**
 - Open the instrument view
 - Zoom in on something
 - Change the x axis range (min/max at the bottom of the screen). 
 - Notice that the zoom is not reset with the fix applied.

Fixes #18250 .

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
